releaseNotes/AzureDevOpsPipelineExtension.md

```diff
@@ -4,6 +4,9 @@ The Azure App Configuration extension in Azure DevOps pipeline has been publishe
 
 ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v3.0.14 - May, 15 2020
+* Fixed the issue that Azure App Configuration pipeline task in version 3.0.13 doesn't load correctly in Linux environment pipeline. [#327](https://github.com/Azure/AppConfiguration/issues/327)
```

Comment:
Is this an issue only in v3.0.13 or is it always the issue? Maybe this?

Fixed the issue that the Azure App Configuration pipeline task does not work in the Linux environment.

Reply:
What @jimmyca15 suggested sounds good to me. Like it.

Reply:
Update the description